pyOSPParser/system_configuration.py

```diff
@@ -124,6 +162,16 @@ class OspInitialValue(OspSystemStructureAbstract):
     _required_keys = ['variable', 'value']
 
     def __init__(self, dict_xml=None, **kwargs):
+        """
+        Constructor for OspInitialValue. One should provide 'variable (str)' and
```

Comment:
I don't understand this comment. Does dict_xml need to include the keys 'variable' and 'value' or should this be called with variable=..., value=...., dict_xml=.... If the later case, why is this not in the included in the signature? Consider also to add a validation with raising a ValueError.

Reply:
You need to provide either dict_xml or a set of arguments: 'variable' and 'value'. This should be an easier sentence to understand. Validation is in the parent class init.